src/Mechanisms/HandleRequests/HandleRequests.php

```diff
@@ -42,9 +42,7 @@ function skipRequestPayloadTamperingMiddleware()
 
     function setUpdateRoute($callback)
     {
-        $route = $callback(function () {
-            return $this->handleUpdate();
-        });
+        $route = $callback([self::class, 'handleUpdate']);
```

Comment:
```suggestion
        $route = $callback($this->handleUpdate(...));
```

Reply:
can you check if that [syntax](https://php.watch/versions/8.1/first-class-callable-syntax) will also work?

Reply:
This does not work

```
#0 [internal function]: Laravel\\SerializableClosure\\Serializers\\Native->__unserialize(NULL)
#1 /Users/sandro/Code/liveiwre-test/vendor/laravel/framework/src/Illuminate/Routing/RouteSignatureParameters.php(22): unserialize('O:55:\"Laravel\\\\S...')
...
```